src/pages/components/callout-quote.mdx

```diff
@@ -23,9 +24,22 @@ The Callout &mdash; quote pattern uses high contrast colors to create a delightf
 
 ![Default](../../images/pattern/callout-quote/callout-quote.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Paragraph | Long copy | Yes   | 1 | LL (200 / 260)|
+| Source (ex. name)  | Short copy | No  | 1 | XS (65 / 85) |
+| Copy 1 (ex. role)  | Short copy | No | 1 | XS (65 / 85) |
+| Copy 2 (ex. company)  | Short copy | No | 1 | XS (65 / 85)   |
+| Quotation mark type | Option list | Yes | - | Double curved (default)<br/>Single curved<br/>Double angle<br/> Single angle<br/> Low high reversed double curved |
+| [CTA (link)](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/cta/) | Component | No | 1 | - |
+
+To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.
```

Comment:
I could be wrong here but I think only one is required here. In fact it appears to be removing the whole attribute in the preview link.

```suggestion
To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer">link</a>.
```

Reply:
@JennySanchez do we have a rule on when to when to open up a new tab using `_blank`? I wonder since because this is link to a page within the site if we shouldn't use `_blank`.

Reply:
@photodow I was doing some research and came across this article: https://pointjupiter.com/what-noopener-noreferrer-nofollow-explained/ . If you think those are unnecessary then we can get rid of them. 

I was thinking since it's out of the carbon for IBM.com site, then we should use _blank, but again open to suggestions.

Reply:
Sorry about the `_blank` I must have been looking at linie 36 or something getting the links confused when I looked in the example page. The lint if I remember correctly was just looking for `noreferrer`. I would say `nofollow` is definitely not needed. I don't think we want to block search engines. I personally don't think any of it is necessary when we can control, trust, and know where we are linking users to. Those attrributes make more sense in a social network scenario in my opinion.